xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlayCodecCCText.h

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (C) 2005-2021 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "DVDOverlayCodec.h"
+#include "DVDStreamInfo.h"
+#include "DVDSubtitles/SubtitlesAdapter.h"
+
+class CDVDOverlay;
+
+class CDVDOverlayCodecCCText : public CDVDOverlayCodec, public CSubtitlesAdapter
+{
+public:
+  CDVDOverlayCodecCCText();
+  ~CDVDOverlayCodecCCText() override;
+  bool Open(CDVDStreamInfo& hints, CDVDCodecOptions& options) override;
+  void Dispose() override;
```

Comment:
This Dispose method always seem to be used internally, please confirm. If so, please make it protected or private

Reply:
yes can be moved to private to all dvdoverlaycodec's